### DISCUSSION TITLE
Borg Backup pauses until AIO Interface is opened

### DISCUSSION THREAD

**Discussion by kemitix** [nextcloud/all-in-one#4467] [Questions] [answered]
My daily scheduled Nextcloud backup is pausing when it starts until I manually open the AIO Interface, at which point it resumes without any further action. It has been doing this every day for at least a week.

The pause has the following lines being logged repeatedly:
```
nc: getaddrinfo for host "nextcloud-aio-apache" port 11000: Name does not resolve                                                                                                              
Waiting for apache to become available 
```
When opening Administration Settings, there is no change.

When opening AIO Interface the backup resumes.

Logs from `nextcloud-aio-mastercontainer` are attached:
- [nextcloud-aio-mastercontainer.stdout.log](https://github.com/nextcloud/all-in-one/files/14804759/nextcloud-aio-mastercontainer.stdout.log)
- [nextcloud-aio-mastercontainer.stderr.log](https://github.com/nextcloud/all-in-one/files/14804760/nextcloud-aio-mastercontainer.stderr.log)

**Comment by szaimen** [comment]
> Borg Backup pauses until AIO Interface is opened

Hi, I fear I cannot reproduce this. I think I have seen the issue in the past already though and usually it was caused by a mountpoint of the borg container as target that was not mounted correctly by the OS...

**Reply by kemitix** [reply]
The mount point is the same as it has been for the last year of successful backups, it is mounted and accessible. There is no timing or scheduling issue around it as it is never unmounted.

**Reply by szaimen** [reply]
Can you post the output of `sudo docker inspect nextcloud-aio-borgbackup` here?

**Reply by kemitix** [reply]
@szaimen Here: [borgbackup.inspect.json](https://github.com/nextcloud/all-in-one/files/14834820/borgbackup.inspect.json)

**Reply by szaimen** [reply]
Apparently you have two bind mounts:
```
...
            {
                "Type": "bind",
                "Source": "/mnt/borg/pcampbell/nextcloud",
                "Destination": "/mnt/borgbackup",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/pcampbell/Docker/nextcloud",
                "Destination": "/host_mounts/home/pcampbell/Docker/nextcloud",
                "Mode": "",
                "RW": false,
                "Propagation": "rprivate"
            },
...
```
Is for example `/mnt/borg` or `/mnt/borg/pcampbell` a network FS like SMB or NFS and lying on a different server in your network?

**Reply by kemitix** [reply]
`/mnt/borg` is a disk to itself. `/home/pcampbell/Docker/nextcloud` is the additional director that is to be included in the backup. Neither are network drives, they are both local disks in the server.

Excerpt from `/etc/fstab` for mount points that cover the two bind mount sources:
```
/dev/mapper/dagon--raid--vg-home /home ext4 defaults 0 2
LABEL=BORG4TB /mnt/borg btrfs defaults,nofail 0 2
```

**Comment by szaimen** [comment, marked as answer]
Hi, this reminds me of https://github.com/nextcloud/all-in-one/discussions/5222

**Reply by kemitix** [reply]
I'll try adding the `mastercontainer` to the `nextcloud-aio`. Hopefully this is a nice simple solution. Thanks.
